google/cloud/aiplatform/docker_utils/local_util.py

```diff
@@ -23,7 +23,9 @@
 _logger = logging.getLogger(__name__)
 
 
-def execute_command(cmd: List[str], input_str: Optional[str] = None) -> int:
+def execute_command(
+    cmd: List[str], input_str: Optional[str] = None, encoding="utf-8", errors=None
```

Comment:
Maybe rename to output_encoding to avoid confusion? Since there's input encoding too.